docs/sources/developers/http_api/reporting.md

```diff
@@ -15,10 +15,315 @@ title: Reporting API
+| layout             | string    | Can be `grid` or `simple`.                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| enableDashboardUrl | bool      | Adds a dashbaord url to the bottom of the report email.                                                                                                                                                                                                                                                                                                                                                                                           |
+| formats            | []string  | Specified what kind of attachment to generate for the report - `csv`, `pdf`, `image`. `pdf` is the default one.                                                                                                                                                                                                                                                                                                                                   |
+| dashboards         | []object  | Dashboards to generate a report for.<br/> [Dashboard object schema](#dashboard-schema).                                                                                                                                                                                                                                                                                                                                                           |
+
+#### Dashboard Schema
+
+| Field name                     | Data type | Description                                              |
+| ------------------------------ | --------- | -------------------------------------------------------- |
+| dashboard.uid                  | string    | Dashboard uid.                                           |
+| timeRange.from                 | string    | Dashboard time range from.                               |
+| timeRange.to                   | string    | Dashboard time range to.                                 |
+| reportVariables.<variableName> | string    | Specifies the name of the dashboard's template variable. |
```

Comment:
@Clarity-89 Could you please take a look at this? Does describing ```[]DashboardDTO `json:"dashboards"``` in this way make sense?